kitsune/community/api.py

```diff
@@ -0,0 +1,330 @@
+from collections import defaultdict
+from datetime import datetime, timedelta
+
+from elasticutils import F
+from rest_framework import views, fields
+from rest_framework.response import Response
+
+from kitsune.questions.models import AnswerMetricsMappingType
+from kitsune.sumo.api import GenericAPIException
+from kitsune.users.models import UserMappingType
+from kitsune.wiki.models import RevisionMetricsMappingType
+
+
+# This should be the higher than the max number of contributors for a
+# section.  There isn't a way to tell ES to just return everything.
```

Comment:
Actually, there is with ElasticUtils--you can do `.everything()`:

https://github.com/mozilla/elasticutils/blob/master/elasticutils/__init__.py#L1566

It does an ES request for the count, then does another ES request to get *count* number of things.

I don't know if that helps you or not.

Also, "This should be *the* higher..." should probably be "This should be higher...".